### PR TITLE
fix(chips): cursor style on static chips

### LINKED ISSUE #3253: Static chips should not have cursor `text`
Since they are static, they can't be edited or even selected, so they should have cursor `default`, not `text`.
This can be seen here:
https://material.angularjs.org/latest/#/demo/material.components.chips
When hovering between chips, a `text` cursor appears.

### PR DESCRIPTION
**Pull request by typotter**
Do not set the cursor style on the md-chips element if there are only static chips.
Also remove some TODO docs that have already been done.

fixes #3253

### CODE REVIEW

**Comment by typotter** [comment]
@jelbourn care to take a look?

File: src/components/chips/chips.scss [hunk 1]
```diff
@@ -55,7 +55,10 @@ $contact-chip-name-width: rem(12) !default;
   font-size: $chip-font-size;
   padding: $chip-wrap-padding;
   vertical-align: middle;
-  cursor: text;
+
+  &:not('.static-only') {
```
Comment (jelbourn) [inline src/components/chips/chips.scss]: Should be `md-static`only`

Also, what do you think of instead naming it `md-readonly` ?

Comment (typotter) [inline src/components/chips/chips.scss]: actually readonly is a good idea. I have another PR that should use this style as well.

**Comment by jelbourn** [comment]
LGTM

**Pull request closed without merging**